tests/test_opds.py

```diff
@@ -199,24 +203,6 @@ def test_lane_url(self):
         feed_url = self.annotator.lane_url(fantasy_lane_without_sublanes)
         eq_(feed_url, self.annotator.feed_url(fantasy_lane_without_sublanes))
 
-    def test_single_entry_no_active_license_pool(self):
```

Comment:
We're not giving a pool to AcquisitionFeed.single_entry at all anymore and the signature has changed since this test was created, so I removed it.